rvps/src/reference_value.rs

```diff
@@ -0,0 +1,197 @@
+// Copyright (c) 2022 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+//! reference value for RVPS
+
+use chrono::{DateTime, NaiveDateTime, Utc};
+use serde::{Deserialize, Deserializer, Serialize};
+
+/// Default version of ReferenceValue
+pub const REFERENCE_VALUE_VERSION: &str = "0.1";
```

Comment:
Same comment as for `MESSAGE_VERSION`: why not use [semver](https://semver.org) here?